zulip_bots/zulip_bots/test_lib.py

```diff
@@ -100,6 +100,17 @@ class StubBotTestCase(TestCase):
 
     bot_name = ''
 
+    def get_response(self, message):
```

Comment:
We probably want to split this out to be the first commit in your PR.  The commit message would be something `bot tests: Add get_response() helper.'